### DISCUSSION TITLE
VDateInput: More flexibility

### DISCUSSION THREAD

**Discussion by zentek-markus** [vuetifyjs/vuetify#19793] [Ideas]
Thanks a lot @johnleider for adding this really nice component. It would be nice to add some more features, especially regarding the behavior of the picker. here are some ideas:
* don't always toggle the picker on field focus (more control when to show the picker)
* more slots and props for controlling picker behavior
* child prop props like pickerProps and inputProps (are these still a thing)?
* better guide on localization / masks / custom formats
* less clicking inside the picker (not alway need to click "ok")
* auto-close the picker on manual input or tab
* more events and handlers for updating/opening/closing the picker

if some of these are already possible, then please update the documentation with examples as i was not able to figure it out easily.

otherwise thanks for your time and effort. :)

**Comment by johnleider** [comment]
1. I'll look into this
2. Will do
3. They are in some areas, what is your usecase?
4. Will do
5. Use the **hide-actions** prop
6. Half of this sounds like a bug, will look into
7. Will think on

**Reply by zentek-markus** [reply]
awesome. thank you very much :D 
as for some example what i am trying to do:
a singular input field w/o outer icons, but both inner icons, left one opens menu A and right one opens menu B, no open-on-focus or whatnot. so far i haven't managed to get this to work properly.... only somewhat... i am also confused about the whole menu parent / accessor triggering...
any help highly appreciated. :)

**Reply by zentek-markus** [reply]
also, please let me know here if there are any significant updates, if possible. :)
